techniques/general/G8.html

```diff
@@ -51,87 +49,83 @@
-  using a more specific technique)
+<a href="https://waic.jp/translations/WCAG22/Understanding/audio-description-or-media-alternative-prerecorded">1.2.3: 音声解説、又はメディアに対する代替 (収録済)</a>
+(より具体的なテクニックを用いれば<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/audio-description-prerecorded">1.2.5: Audio Description (Prerecorded)</a>
-(<strong>Sufficient</strong>
-  using a more specific technique)
+<a href="https://waic.jp/translations/WCAG22/Understanding/audio-description-prerecorded">1.2.5: 音声解説 (収録済)</a>
+(より具体的なテクニックを用いれば<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/extended-audio-description-prerecorded">1.2.7: Extended Audio Description (Prerecorded)</a>
-(<strong>Sufficient</strong>
-  using a more specific technique)
+<a href="https://waic.jp/translations/WCAG22/Understanding/extended-audio-description-prerecorded">1.2.7: 拡張音声解説 (収録済)</a>
+(より具体的なテクニックを用いれば<strong>十分なテクニック</strong>)
     </li>
   
   </ul>
   
 
 
 
-  <p>This technique applies to any technology that supports audio and video.</p>
+  <p>このテクニックは、音声及び映像をサポートする全ての技術に適用される。</p>
 
 </div>
 </section>
 	
 
-<section id="description"><h2>Description</h2>
-      <p>The objective of this technique is to provide a second version of video content that provides extended audio descriptions in the same <a href="https://w3c.github.io/wcag/guidelines/22/#dfn-human-language-s" target="terms">human language</a> as the video. One of the difficult things about creating traditional audio descriptions is that the narrator sometimes has to provide a lot of information during very short pauses in dialogue. Extended audio description temporarily pauses the audio and video to allow critical information to be delivered when pauses in dialogue are insufficient for adequate description.</p>
-      <p>Providing a second version of the movie with extended audio descriptions will make this content accessible for blind people who need to hear not only the dialogue but also the context and other aspects of the video that are not communicated by the characters' dialogue alone, and for which there is insufficient time during the natural dialogue.</p>
-      <p>Because it disrupts viewing for those who do not need the additional description, techniques that allow you to turn the feature on and off are often provided. Alternately, versions with and without the additional description can be provided.</p>
-   </section><section id="examples"><h2>Examples</h2>
-      <section class="example" id="example-1"><h3>Example 1</h3>
+<section id="description"><h2>解説</h2>
+      <p>このテクニックの目的は、映像コンテンツに対して映像と同じ<a href="https://w3c.github.io/wcag/guidelines/22/#dfn-human-language-s" target="terms">自然言語</a>で拡張した音声解説のある二つめのバージョンを提供することである。一般的な音声解説の作成に伴う困難の一つとして、非常に短い発話の合間に、たくさんの情報をナレーションで提供しなければならない状況が時々生じることが挙げられる。発話の止まる合間が適切な音声解説を提供するのに十分でない場合は、拡張した音声解説を提供するために音声と映像を一時停止して、重要な情報を伝えられるようにする。</p>
```

Comment:
extended audio description は WCAG の用語で「拡張音声解説」なので、訳を調整しました。
```suggestion
      <p>このテクニックの目的は、映像コンテンツに対して、映像と同じ<a href="https://w3c.github.io/wcag/guidelines/22/#dfn-human-language-s" target="terms">自然言語</a>による拡張音声解説のある、二つめのバージョンを提供することである。一般的な音声解説の作成に伴う困難の一つとして、非常に短い発話の合間に、たくさんの情報をナレーションで提供しなければならない状況が時々生じることが挙げられる。発話の止まる合間が適切な音声解説を提供するのに十分でない場合は、拡張した音声解説を提供するために音声と映像を一時停止して、重要な情報を伝えられるようにする。</p>
```